openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/AddDataQualityTestV1.tsx

```diff
@@ -255,13 +236,13 @@ const AddDataQualityTestV1: React.FC<AddDataQualityTestProps> = ({ table }) => {
         <SuccessScreen
           handleIngestionClick={() => setAddIngestion(true)}
           handleViewServiceClick={handleViewTestSuiteClick}
-          isAirflowSetup={isAirflowRunning}
+          isAirflowSetup={isAirflowAvailable}
```

Comment:
Can't we use hook inside SuccessScreen directly?

Reply:
I will check and make the changes.